### DISCUSSION TITLE
Akka.DependencyInjection doesn't create actor when can't resolve dependencies but it doesn't throw error

### DISCUSSION THREAD

**Discussion by lnodaba** [akkadotnet/akka.net#5727] [Q&A] [answered]
Hello!

I'm using the DI in the following way:

```
 _actorSystem.ActorOf(Supervise(PropsWithDI<MessagePublisherActor>()), MessagePublisher.ActorName);

private Props PropsWithDI<T>(params object[] args) where T : ActorBase
       => DependencyResolver.For(_actorSystem).Props<T>(args);

```

If somehow there is a Constructor parameter which isn't added to the DI container, it simply doesn't create the Actor, which is normal of course but I would expect an error or something.

Am I using it wrong or is this a missing functionality of Akka.NET Akka.DependencyInjection.

**Comment by Aaronontheweb** [comment, marked as answer]
Is this the same issue @lnodaba https://github.com/akkadotnet/akka.net/issues/5734 ?

**Reply by lnodaba** [reply]
Yes it is.

**Comment by lnodaba** [comment]
Found out a solution for catching the error, gave an answer to the other issue: https://github.com/akkadotnet/akka.net/issues/5734#issuecomment-1135829709
